tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "outDir": "intermediate",
```

Comment:
yeah, I'm not sure what we should use. Intermediate isn't standard, but I thought that it would be weird having build/runtime/build.

This is pretty easy to tweak later, shall we leave it as this for now?